test/unit/proxy_statuses/proxy_status_tftp.rb

```diff
@@ -0,0 +1,42 @@
+require 'test_helper'
+
+class ProxyStatusTftp < ActiveSupport::TestCase
```

Comment:
This test is not being run, the file needs to end in `_test`, and the class has to be named `ProxyStatusTftpTest`